.github/workflows/build.yml

```diff
@@ -0,0 +1,42 @@
+name: Build
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: windows-2019
+    steps:
+    # RTCV
+    - name: Checkout RTCV peer dependency
+      uses: actions/checkout@v2
+      with:
+        repository: 'ircluzar/RTCV'
+        ref: '506'
```

Comment:
If we do the branch architecture switch, this may need to be `stable` or `beta` or `main`, or be a variable based on the current branch that's being built or PR'd into

Reply:
Sounds good. We'll have to do something of the sort so we'll figure it out once we execute on that.